src/NuGet.Clients/NuGet.PackageManagement.UI/Options/ConfigPathsOptionsPage.cs

```diff
@@ -0,0 +1,55 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+using System;
+using System.ComponentModel;
+using System.Runtime.InteropServices;
+using System.Threading;
+using System.Windows;
+using Microsoft.VisualStudio.Shell;
+
+namespace NuGet.PackageManagement.UI.Options
+{
+    [Guid("C17B308A-00BB-446E-9212-2D14E1005985")]
+
+    public class ConfigPathsOptionsPage : UIElementDialogPage
+    {
+        private Lazy<ConfigPathsControl> _configPathsControl;
+
+        /// <summary>
+        /// Gets the Windows Presentation Foundation (WPF) child element to be hosted inside the Options dialog page.
+        /// </summary>
+        /// <returns>The WPF child element.</returns>
+        protected override UIElement Child => _configPathsControl.Value;
+
+        public ConfigPathsOptionsPage()
+        {
+            _configPathsControl = new Lazy<ConfigPathsControl>(() => new ConfigPathsControl());
```

Comment:
What about creating a new instance of ViewModel for the control and passing it in the View's constructor?